ses/domain/main.tf

```diff
@@ -26,6 +26,16 @@ resource "aws_route53_record" "ses_verification" {
   records = [aws_ses_domain_identity.domain[0].verification_token]
 }
 
+resource "aws_route53_record" "txt" {
+  count = var.create && length(var.txt_records) > 0 ? 1 : 0
+
+  zone_id = var.hosted_zone_id
+  name    = "${local.domain}."
+  type    = "TXT"
```

Comment:
@mskrajnowski 
Although I'm not sure if Terraform will merge both `TXT` resources for the same domain 🤔 

Do you have some way of testing/generating an example plan for development version of terraform-modules?

Reply:
I'm pretty sure this will fail, because AWS won't allow two records with the same name and type.

As for testing, I was just deploying the examples to test if they work properly, I don't know of a better way to test terraform than to actually create the infrastructure and check if it was created properly.

Reply:
>I'm pretty sure this will fail, because AWS won't allow two records with the same name and type.

That's what I was afraid of, so perhaps we need something like https://github.com/codequest-eu/terraform-modules/pull/31/commits/53f7b2ea97e0e0cf995a21a242998af0816300fb